From 64018156d2ec31352aa765519028e7eb5858923b Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Wed, 2 Dec 2009 13:00:24 -0500
Subject: [PATCH 10/43] vbi: vbiCtx_t --> int32_t

for i in `git grep -l vbiCtx_t` ; do mv $i $i~ ; cat $i~|sed 's/vbiCtx_t/int32_t/' > $i ; done

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 include/vbi/compat.h  |    1 +
 include/vbi/syscall.h |    6 +++---
 include/vbi/types.h   |    1 -
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/include/vbi/compat.h b/include/vbi/compat.h
index 77e68d7..4bf1d0d 100644
--- a/include/vbi/compat.h
+++ b/include/vbi/compat.h
@@ -49,6 +49,7 @@
 #define vbiIrq_t		uint32_t
 #define vbiVb_t			uint32_t
 #define vbiCore_t		int32_t
+#define vbiCtx_t		int32_t
 
 /*
  * struct fields and similar
diff --git a/include/vbi/syscall.h b/include/vbi/syscall.h
index 276e376..571261b 100644
--- a/include/vbi/syscall.h
+++ b/include/vbi/syscall.h
@@ -32,7 +32,7 @@ struct vmmuConfig;
 
 typedef struct vbiMsgInfo
 {
-	vbiCtx_t    id;	/* context id of sender */
+	int32_t    id;	/* context id of sender */
 	uint32_t type;	/* message type (msg/event) */
 	size_t   slen;	/* length of sent buffer */
 	size_t   rlen;	/* length of received buffer */
@@ -99,14 +99,14 @@ extern asmlinkage int vbi_vtlb_op(unsigned int op, unsigned long arg1,
  */
 
 /* Message send */
-extern asmlinkage int32_t vbi_send(vbiCtx_t id, void *smsg, size_t slen, 
+extern asmlinkage int32_t vbi_send(int32_t id, void *smsg, size_t slen, 
 			    void *rmsg, size_t rlen, VBI_MSG_INFO *info, 
 			    VBI_MSG_CTL *ctl);
 /* message receive */
 extern int32_t vbi_receive(void *rmsg, uint32_t rlen, VBI_MSG_INFO *info,
 				VBI_MSG_CTL *ctl);
 /* message reply */
-extern asmlinkage int32_t vbi_reply(vbiCtx_t id, void *smsg, size_t slen, 
+extern asmlinkage int32_t vbi_reply(int32_t id, void *smsg, size_t slen, 
 				VBI_MSG_CTL *ctl);
 
 extern asmlinkage int32_t vbi_panic(const char *msg);
diff --git a/include/vbi/types.h b/include/vbi/types.h
index 3de0908..9f2853d 100644
--- a/include/vbi/types.h
+++ b/include/vbi/types.h
@@ -34,7 +34,6 @@
 
 #ifndef _ASMLANGUAGE
 
-typedef int32_t vbiCtx_t;
 typedef uint64_t vbiPhysAddr_t;
 typedef void *vbiGuestPhysAddr_t;
 typedef int32_t vbiIntState_t;
-- 
1.6.5.2

